src/components/atoms/button/stories/button.stories.common.js

```diff
@@ -1,23 +1,47 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react-native';
 import { action } from '@storybook/addon-actions';
-import { Text, View } from 'react-native';
+import { Text, View, Switch } from 'react-native';
+import { css } from '@emotion/native';
+import { useTheme } from 'emotion-theming';
 import { SafeView } from 'storybook/story-components-native';
+import { ThemeProvider, ThemeActionsContext } from 'store';
 
 import Button from '../';
 
+const ThemeSwitcher = () => {
+  const { toggleTheme } = React.useContext(ThemeActionsContext);
+  const theme = useTheme();
+  return (
+    <View
+      style={css`
```

Comment:
Is it okay to create style objects during render like this in Emotion? In Styled Components it's something to avoid and can creating constantly-expanding stylesheets that can cause slowdowns if the app is running a long time. 

We're not using anything from the function's scope here, so this `css` call could be a static variable defined outside `ThemeSwitcher`? Then the function is only called once.